### PR TITLE
update contributing doc.

### PR DESCRIPTION
**Pull request by morvencao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/132

**Specify your PR reviewers**:

/cc @gyliu513 @jichenjc @xunpan

**Special notes for your reviewer**:

/sig ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,29 +14,27 @@ Kubernetes projects require that you sign a Contributor License Agreement (CLA)
...
+export REGISTRY=<your-docker-registry>
+make build-push-images
 ```
 
 ## Start kind and deploy the provider
 
 ```shell
 go get -u sigs.k8s.io/kind && kind create cluster
 export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+make install
```
Comment (gyliu513) [inline CONTRIBUTING.md]: Can you please add some command line here to help check if the controllers and pods are running?

Comment (morvencao) [reply CONTRIBUTING.md]: Add lint to guide in README.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,19 +29,4 @@ export REGISTRY=<your-docker-registry>
...
-
-## Verify deployment
-
-TODO: Should deploy a sample Cluster and Machine resource to verify controllers are reconciling properly. This is troublesome however since before the actuator stubs are filled in, all we will see is messages to the effect of "TODO: Not yet implemented"..."
-
-```
-kubectl logs cluster-api-provider-ibmcloud-controller-manager-0 -n cluster-api-provider-ibmcloud-system
-```
\ No newline at end of file
+Now, you can follow the [guide](./README.md#getting-started) to get started with the Cluster API Provider IBM Cloud.
```
Comment (gyliu513) [inline CONTRIBUTING.md]: s/guide/getting started guide

Comment (gyliu513) [reply CONTRIBUTING.md]: s/to get started with/to get start with

**Pull request merged**